x/ccv/provider/client/proposal_handler.go

```diff
@@ -172,6 +173,70 @@ Where proposal.json contains:
 	}
 }
 
+// SubmitConsumerModificationProposalTxCmd returns a CLI command handler for submitting
+// a consumer modification proposal via a transaction.
+func SubmitConsumerModificationProposalTxCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "consumer-modification [proposal-file]",
+		Args:  cobra.ExactArgs(1),
+		Short: "Submit a consumer modification proposal",
+		Long: `
+Submit a consumer modification proposal along with an initial deposit.
+The proposal details must be supplied via a JSON file.
+
+Example:
+$ <appd> tx gov submit-legacy-proposal consumer-modification <path/to/proposal.json> --from=<key_or_address>
+
+Where proposal.json contains:
+
+{
+    "title": "Modify FooChain",
+    "summary": "Make it an Opt IN chain",
```

Comment:
```suggestion
    "summary": "Make it an Opt In chain",
```
unless the capitalization was for emphasis?